src/components/Main/Header.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import Logo from '../../images/logo.png';
+
+function Header () {
+  return (
+        <header className="header-section">
+            <div className="container">
+                <div className="inner-header">
+                    <div className="row">
+                        <div className="col-lg-2 col-md-2">
+                            <div className="logo">
+                                <a href="#">
```

Comment:
We don't use <a> in React. We need to use react-router and it's <Route path="..."> component